plugins/network-elements/nuage-vsp/src/net/nuage/vsp/acs/client/NuageVspApiClient.java

```diff
@@ -21,10 +21,10 @@
 
 public interface NuageVspApiClient {
 
-    public void login() throws Exception;
+    void login() throws Exception;
```

Comment:
This is how the methods are declared in our client. They need to have the same signature as defined in our client.